### PR TITLE
Update regions.json

### PR DESCRIPTION
**Pull request by freearhey**
- added missing countries to the MENA region
- added [Northern America](https://en.wikipedia.org/wiki/Northern_America) region (Resolves #5624)
- added [Latin America and the Caribbean](https://en.wikipedia.org/wiki/Latin_America_and_the_Caribbean) region

### CODE REVIEW

**Review by Sphinxroot — APPROVED** [review]

File: scripts/data/regions.json [hunk 1]
```diff
@@ -561,10 +612,12 @@
     "country_codes": [
       "AE",
       "BH",
+      "CY",
       "DJ",
       "DZ",
       "EG",
       "EH",
+      "IL",
```
Comment (LaneSh4d0w) [inline scripts/data/regions.json:620]: Israel ? Not quite sure about this one.

File: scripts/data/regions.json [hunk 2]
```diff
@@ -561,10 +612,12 @@
     "country_codes": [
       "AE",
       "BH",
+      "CY",
```
Comment (LaneSh4d0w) [inline scripts/data/regions.json:615]: Cyprus ? Not quite sure about that one.

File: scripts/data/regions.json (hunk 1 above)
Comment (Sphinxroot) [reply scripts/data/regions.json:620]: https://www.iban.com/country-codes

Comment (LaneSh4d0w) [reply scripts/data/regions.json:620]: Yes, that confirms what I was saying. IL is Israel, it is not a country around the LATAM region, but on the Middle East one.

Comment (freearhey) [reply scripts/data/regions.json:620]: Please check again. These changes have nothing to do with the LATAM region.

**Review by BellezaEmporium — APPROVED** [review]

**Pull request merged**